pb_app/lib/submission_form.dart

```diff
@@ -112,33 +121,21 @@ class _Card extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Card(
-      margin: const EdgeInsets.all(25),
+      margin: const EdgeInsets.symmetric(
+        vertical: 25,
+        horizontal: 5,
+      ),
       color: Colors.black38,
       shape: platformAwareShape(50),
       child: InkWell(
-        onTap: () {},
+        onTap: () {
+          // TODO: InkWell not working properly into stack (https://github.com/flutter/flutter/issues/104519)
```

Comment:
This issue was resolved in https://github.com/lukepighetti/pocketbase_flutter_app/commit/e374c182affd4a2debd5b4b0a1647de493d78685 so the comment should be removed